### ISSUE TITLE
why sudo?

### ISSUE THREAD

**Issue by aaronsql2019** [myvesta/vesta#187]
when I use the myVesta Configuration tool, the FIRST WORD is 'sudo'. Why? You specifically target debian people.

I tend to think of sudo as an ubuntu invention. 
I don't use it. I login as root. 

On UBUNTU I use Sudo. 
From what I see, sudo is ONLY enabled on debian if we CHOOSE a blank password for the 'root' account during setup.

I didn't KNOW that was an option. I don't know why ANYONE would ever choose to do that.

Example:
**sudo** bash vst-install-debian.sh --apache yes --nginx yes --phpfpm no --vsftpd no --proftpd yes --named yes --mysql yes --mysql8 no --postgresql no --exim yes --dovecot yes --clamav yes --spamassassin yes --iptables yes --fail2ban yes --quota no --force no --interactive yes --port 8083 --password myvestapassword --lang en 
 
What I should that it should be
bash vst-install-debian.sh --apache yes --nginx yes --phpfpm no --vsftpd no --proftpd yes --named yes --mysql yes --mysql8 no --postgresql no --exim yes --dovecot yes --clamav yes --spamassassin yes --iptables yes --fail2ban yes --quota no --force no --interactive yes --port 8083 --password myvestapassword --lang en

**Comment by mckaygerhard** [comment]
i agree.. why not **doas** or **doas-sudo-shim** ? i opened same issue in OMV project but those people arged "standar" that all we know was impossed by company managed distros like feladora or winbuntu

i currently changed to doas usinig a package based on **doas-sudo-shim**, and with **doas-sudo-shim** prviding "sudo" (not doas/opendoas per ser the provider of sudo) and works like a charm.. 

i used the package: https://build.opensuse.org/package/show/home:vegnuli:system-vnx2/sudo that is really the **doas-sudo-shim** as sudo crap

currently the myvesta doe snot used any GUI advanced feature from the windolike sudo so change to doas+doassudoshim is not hard
